components/normalizer/trie-value-format.md

```diff
@@ -0,0 +1,42 @@
+# The `icu_normalizer` Trie Value Format
```

Comment:
Nit: mention this markdown file in all docs for values with the u32.

Reply:
Done.